pkg/resources/validation/namespace/namespace.go

```diff
@@ -38,17 +40,63 @@ func (v *Validator) GVR() schema.GroupVersionResource {
 }
 
 // Operations returns list of operations handled by this validator.
-func (v *Validator) Operations() []admissionregistrationv1.OperationType {
```

Comment:
I don't think that this makes sense as part of the interface any longer. Each `adm1.ValidatingWebhook` can cover it's own set of operations, and there's no reason why they need to share the same types of operations. 

In this implementation, which is the first actual use of using this interface to produce multiple  `adm1.ValidatingWebhook`s, we produce two different webhooks with two different sets of operations. If this produces some off logic in the very first use case, I think it's better to reconsider it.